### ISSUE TITLE
--config-file ?

### ISSUE THREAD

**Issue by pouledodue** [rambler-digital-solutions/scaraplate#8]
hi is there an equivalent to cookicutter's `--config-file` ?

**Comment by KostyaEsmukov** [comment]
No.

As per https://cookiecutter.readthedocs.io/en/latest/advanced/user_config.html , the following can be specified in the `--config-file`:
- `cookiecutters_dir`/`abbreviations`. git templates are not supported by scaraplate (currently). We cannot just let cookiecutter handle them, because scaraplate needs to access the templates too to read scaraplate.yaml.
- `default_context` doesn't make any sense for non-initial rollups (as context is provided by [CookieCutterContext](https://scaraplate.readthedocs.io/en/latest/template.html#module-scaraplate.cookiecutter)), and for initial rollup (which is a one-time manual operation) the variables need to be specified manually anyways.
- `replay_dir` is explicitly mocked out by scaraplate, because template variables should be retrieved solely from the files in the target projects, for the rollups to be reproducible.

I'm closing this issue, but feel free to continue the thread if you have further questions.

**Comment by pouledodue** [comment]
is there a way to pass extra_context variables? like `cookiecutter ./template_yoshi full_name="fd"`

**Comment by KostyaEsmukov** [comment]
Added in #10, will be released as 0.2.

In the meantime this can be tested by installing from the master branch directly:

```
pip install 'git+https://github.com/rambler-digital-solutions/scaraplate.git'
```
